espanol/canudos.md

```diff
@@ -154,3 +154,6 @@ Y llego [Daynerys Targaryen](thrones/daynerys.md)
 
 Recordó algunos crímenes que pesaban sobre él y de la nada perdió el conocimiento(desmayo/desmayo.md)
 
+Revelo que venia del futuro y que le daba dos opciones: 
+- [Morir luchando como sabia que pasaria en el futuro](time/time.md)
+- [Seguir sus instrucciones y ganar](time/time1.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.